### DISCUSSION TITLE
Kitty and Textual Application

### DISCUSSION THREAD

**Discussion by tgu-ltd** [kovidgoyal/kitty#7277] [Show and tell]
Hi, Very quick message to say thanks to every one involved in developing Kitty and hopefully the recently published [TGUTUI demonstration application](https://github.com/tgu-ltd/tgutui) does it justice.

**Comment by kovidgoyal** [comment]
Cool, gives me undergrad physics lab vibes :)
Linked in the kitty integrations doc:
https://github.com/kovidgoyal/kitty/commit/a695b4ebe18a351513cb4bf4eaf2f170f78c76cf
